utils/object.js

```diff
@@ -56,7 +56,9 @@ export function get(obj, path) {
 
   let parts;
 
-  if ( path.match(quotedKey) ) {
+  if (Array.isArray(path)) {
+    parts = path;
```

Comment:
You can already do keys that might contain dots with ```get(this, `metadata.annotations."${label}"`);```, I don't think a new array syntax is necessary.